### ISSUE TITLE
can we link it to fake address generator site or any alter? native regarding it

### ISSUE THREAD

**Issue by mycal123** [nativ18/AutoCompleteChromePlugin#1]
honestly i was looking for any official extension of any fake address generator site. coincidentally found this extension.

**Comment by realNativLevy** [comment]
I don't see how this extension can be used as a fake address generator. It only memorizes the user's past sentences and suggest it to him in future discussions.

**Comment by mycal123** [comment]
true. can you suggest any other third-party extensions which fulfill what i have asked?
